### PR TITLE
debian12.2-slim, latest rspamd and gucci dependancies

### LINKED ISSUE #22: rspamd 3.5 ratelimit issues, bundled in debian-mail-overlay image
Hello. we recently noticed a weird behaviour with rspamd  ratelimits once switching to latest mailserver image (we were not on previous one, but on older one shipping old 3.4 rspamd. 

after some searches, a similar issue was raised in other mailstacks like mailcow : https://github.com/mailcow/mailcow-dockerized/issues/5168

they got exact similar issues when switching to rspamd 3.5

instead of bundling a static version from github repo, why not use rspamd repos? they propose the sources in their download page, section debian / ubuntu : https://rspamd.com/downloads.html

also, proposing a newer debian mail overlay based on debian12.2-slim could help switch to latest 
- clamav bundled in bookworm, which is now 1.0.3  (instead of 1.0.1) https://packages.debian.org/bookworm/clamav
- postfix bundled in bookworm which is now 3.7.6 (instead of 3.7.5-2) https://packages.debian.org/bookworm/postfix

### PR DESCRIPTION
**Pull request by diroots**
fix https://github.com/mailserver2/debian-mail-overlay/issues/22

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,10 +4,10 @@ This overlay base image contains Debian 11 "Bullseye" slim (remove some extra fi
 
 Software built from source :
 
-* Skalibs 2.13.1.1 : <https://skarnet.org/software/skalibs/>
-* Execline 2.9.3.0 : <https://skarnet.org/software/execline/>
-* s6 2.11.3.2 : <https://skarnet.org/software/s6/>
-* Rspamd 3.5 : <https://rspamd.com/>
-* Gucci 1.6.6 : <https://github.com/noqcks/gucci/>
+* Skalibs 2.14.0.1 : <https://skarnet.org/software/skalibs/>
```
Comment (AndrewSav) [inline README.md:7]: This does not match the version in Dockerfile, which is `2.13.1.1`

File: README.md [hunk 2]
```diff
@@ -4,10 +4,10 @@ This overlay base image contains Debian 11 "Bullseye" slim (remove some extra fi
...
+* Execline 2.9.4.0 : <https://skarnet.org/software/execline/>
```
Comment (AndrewSav) [inline README.md:8]: This does not match the version in Dockerfile, which is 2.9.3.0

File: README.md [hunk 3]
```diff
@@ -4,10 +4,10 @@ This overlay base image contains Debian 11 "Bullseye" slim (remove some extra fi
...
+* s6 2.12.0.2 : <https://skarnet.org/software/s6/>
```
Comment (AndrewSav) [inline README.md:9]: This does not match the version in Dockerfile, which is 2.11.3.2

**Review by AndrewSav — CHANGES_REQUESTED** [review]

**Comment by AndrewSav** [comment]
Looking good to me, let's get these versions sorted, then I'll run it for a week with the new image (we will also need a mailserver PR for that I'm guessing) and then we can merge.

**Comment by diroots** [comment]
i did update the dockerfile so all versions are aligned @AndrewSav

**Comment by diroots** [comment]
ok i think i know why the 3 skarnet dependancies were not updated in the dockerfile, but in the readme : i did run the 'update-dependencies.sh script with bash instead of zsh and it could not do the string replacements in the dockerfile, it did only in the readme file. fully fixed now

**Comment by AndrewSav** [comment]
@diroots thank you for this. Have you tested this? If so I think I'll just merge it without testing because we can test it when we do the PR for the mailserver repo, but I'd like to know if you ran it and it worked for you

**Review by AndrewSav — APPROVED** [review]

**Comment by diroots** [comment]
i did not run this as is yet. i did update postfix and clamav from with container to the latest version from debian repo, and i did test the update to rspamd 3.7.4 with adding the repo in the container and install it from there. 

all seemed to work when restarting container with these versions

**Comment by diroots** [comment]
@AndrewSav i would also say it 'safe' to merge, so it could biuld the underlying debian-mail-overlay image. 

then we could test to spin up a mailserver container based on that, on a test env / branch

**Comment by AndrewSav** [comment]
@diroots we are building/pushing the image manually. I'll try to find time to work on this over the weekend

**Comment by diroots** [comment]
@AndrewSav 
- i did build a mailserver based on this new image. 
- i did deploy this mailserver on some of our staging env. upgrade from previous 1.1.15 version went seamlessly, and the stack is running as expected. i can see updated rspamd/postfix/clamav. all these service logs show it runs smoothly as it was before.

**Pull request merged**